osquery/CMakeLists.txt

```diff
@@ -98,6 +98,9 @@ ADD_OSQUERY_LINK_CORE("glog")
 ADD_OSQUERY_LINK_CORE("boost_system")
 ADD_OSQUERY_LINK_CORE("boost_filesystem")
 ADD_OSQUERY_LINK_CORE("boost_regex")
+ADD_OSQUERY_LINK_CORE("cryptsetup")
```

Comment:
Yeah, if you don't mind. I'd like to see the linking errors. There shouldn't be anything in osquery "core" (essentially the non-table, non-plugin) code that requires `libcryptsetup` symbols.